tests/components/rfxtrx/test_config_flow.py

```diff
@@ -1,14 +1,282 @@
 """Test the Tado config flow."""
+import os
+
+import serial.tools.list_ports
+
 from homeassistant import config_entries, data_entry_flow, setup
-from homeassistant.components.rfxtrx import DOMAIN
+from homeassistant.components.rfxtrx import DOMAIN, config_flow
 from homeassistant.helpers.device_registry import (
     async_entries_for_config_entry,
     async_get_registry,
 )
 
+from tests.async_mock import MagicMock, patch, sentinel
 from tests.common import MockConfigEntry
 
 
+def serial_connect(self):
+    """Mock a serial connection."""
+    self.serial = True
+
+
+def serial_connect_fail(self):
+    """Mock a failed serial connection."""
+    self.serial = None
+
+
+def com_port():
+    """Mock of a serial port."""
+    port = serial.tools.list_ports_common.ListPortInfo()
+    port.serial_number = "1234"
+    port.manufacturer = "Virtual serial port"
+    port.device = "/dev/ttyUSB1234"
+    port.description = "Some serial port"
+
+    return port
+
+
+@patch(
+    "homeassistant.components.rfxtrx.rfxtrxmod.PyNetworkTransport.connect",
+    MagicMock(return_value=None),
+)
+async def test_setup_network(hass):
```

Comment:
I think we need to accept the decorator patched mock as a parameter here.

Reply:
Yes, that solves it, thanks!